Formula/macvim.rb

```diff
@@ -14,20 +14,29 @@ class Macvim < Formula
   end
 
   option "with-override-system-vim", "Override system vim"
+  option "with-gettext", "Build MacVim with +gettext."
 
   deprecated_option "override-system-vim" => "with-override-system-vim"
 
+  conflicts_with "vim",
+    :because => "They both override system vim" if build.with? "override-system-vim"
```

Comment:
This won't work properly unless you also inspect the `tab` (INSTALL_RECEIPT) of vim to see whether it was built with `override-system-vi`.